src/mlpack/methods/kde/kde_model_impl.hpp

```diff
@@ -352,6 +492,11 @@ void KDEModel::serialize(Archive& ar, const unsigned int /* version */)
   ar & BOOST_SERIALIZATION_NVP(absError);
   ar & BOOST_SERIALIZATION_NVP(kernelType);
   ar & BOOST_SERIALIZATION_NVP(treeType);
+  ar & BOOST_SERIALIZATION_NVP(monteCarlo);
+  ar & BOOST_SERIALIZATION_NVP(MCProb);
+  ar & BOOST_SERIALIZATION_NVP(initialSampleSize);
+  ar & BOOST_SERIALIZATION_NVP(MCEntryCoef);
+  ar & BOOST_SERIALIZATION_NVP(MCBreakCoef);
```

Comment:
Is it necessary to serialize these, if they are already held in the `KDE` object?

Reply:
I think it's a good idea since it makes possible to load a model and get the value of those variables without having to implement visitors for each of them right?